.github/workflows/build.yml

```diff
@@ -36,7 +36,7 @@ jobs:
                     echo $extension_version >> $env:GITHUB_ENV
                 - name: Setup PHP
                   id: setup-php
-                  uses: php/setup-php-sdk@v0.8
+                  uses: php/setup-php-sdk@v0.10
```

Comment:
@cmb69 hmm, would you be able to point me to what might be going wrong in CI here? I tried updating `setup-php-sdk` (taking inspiration from php/pecl-file_formats-yaml#89) but still getting failures for Win builds on PHP 7.3 and 7.4 - do these older PHP versions need something different? :thinking: 

```
Run php/setup-php-sdk@v0.10
Run D:\a\_actions\php\setup-php-sdk\v0.10/determine-revision -version 7.4
Run D:\a\_actions\php\setup-php-sdk\v0.10/run -version 7.4 -revision 7.4.33 -baseurl https://downloads.php.net/~windows/releases/archives -arch x64 -ts ts -deps @()
no suitable toolset available on this runner
At D:\a\_actions\php\setup-php-sdk\v0.10\run.ps1:47 char:5
+     throw "no suitable toolset available on this runner"
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : OperationStopped: (no suitable too... on this runner:String) [], RuntimeException
    + FullyQualifiedErrorId : no suitable toolset available on this runner
 
Error: Process completed with exit code 1.
```

example failing job: https://github.com/php/pecl-datetime-timezonedb/actions/runs/11252496685/job/31285730741?pr=12

Reply:
The action runs on windows-latest, that is currently windows-2022. However, these runners no longer have support for building with msv141. The simplest solution is to let the action run on windows-2019 for now (until PHP-8.4 is required in which case a [matrix exclusion](https://github.com/php/setup-php-sdk?tab=readme-ov-file#inputs) is the best solution).

Reply:
Thank you so much @cmb69 that worked perfectly :100: ! I went with matrix exclusion, as I assume PHP 8.4 will be added pretty soon!